style.css

```diff
@@ -0,0 +1,193 @@
+body {
+    margin: 0;
+    font-family: 'Montserrat', sans-serif;
+    font-size: 14px;
+    font-style: normal;
+    font-weight: 600;
+    line-height: 24px;
+    letter-spacing: 0.2px;
+}
+
+/* add backgound image and place content in center */
+header {
+    background-image: linear-gradient(rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5)), url(img/coverback-image.png);
+    background-size: cover;
+    box-sizing: border-box;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+}
+
+
+
+/* STYLE NAVIGATION BAR: LOGO AND RIGHT SIDE OF THE BAR  */
+
+nav {
+    display: flex;
+    align-items: center;
+    justify-content: space-around;
+    gap: 462px;
+    margin: 0px 59px 0px 59px;
+    padding: 23px 164px 23px 136px;
+    max-width: 1322px;
+}
+
+.nav-right-side {
+    display: flex;
+    flex-direction: row;
```

Comment:
no need to set flex-direction: row if it wasn't set to column before. 
As it is row by default anyway